ReactWindows/ReactNative.Shared/IAsyncCancelable.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// Asynchronous disposable resource with disposal state tracking.
+    /// </summary>
+    public interface IAsyncCancelable : IAsyncDisposable
+    {
```

Comment:
Looks like you have a bunch of files at the top level of ReactNative.Shared that are not being used.